src/emit/substrate/target.rs

```diff
@@ -1062,8 +1068,9 @@ impl<'a> TargetRuntime<'a> for SubstrateTarget {
         .left()
         .unwrap()
         .into_int_value();
-
-        log_return_code(binary, "seal_debug_message", ret);
+        if !is_error {
```

Comment:
Why did you add this guard?

We must assume that `seal_debug_message` can fail too. But this guard hides it from the user.